### PR TITLE
Foreign activities benefits

### PR DESCRIPTION
**Pull request by floresj**

### CODE REVIEW

File: src/components/Section/Foreign/Activities/BenefitActivity/Benefit.jsx [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+              onChange={this.updateBenefitType}
+              onValidate={this.props.onValidate}
+            />
+            <Radio name="benefit_type"
+              label={i18n.m('foreign.activities.benefit.label.benefitTypes.other')}
+              value="Other"
+              onChange={this.updateBenefitType}
+              onValidate={this.props.onValidate}
+            />
+          <Show when={this.props.BenefitType === 'Other'}>
```
Comment (bmallred) [inline src/components/Section/Foreign/Activities/BenefitActivity/Benefit.jsx]: The `Show` may be better placed outside of the `RadioGroup`

File: src/components/Section/Foreign/Activities/BenefitActivity/BenefitActivity.jsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    return prev
+  })
+
+  return (
+    <span className="content">
+      <span className="index">{type}: {index + 1}</span>
+      <span className="benefit-summary">
+        <strong>{ summary || i18n.t('foreign.activities.benefit.collection.summary')}</strong>
+      </span>
+      <span className="date">{benefit.Date}</span>
```
Comment (bmallred) [inline src/components/Section/Foreign/Activities/BenefitActivity/BenefitActivity.jsx]: Does this need to be wrapped in `<strong>`?

File: src/components/Section/Foreign/Activities/BenefitActivity/ContinuingBenefit.jsx [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+            onValidate={this.props.onValidate}
+          />
+        </Field>
+
+        <Branch name="Obligated"
+          className="obligated"
+          label={i18n.t('foreign.activities.benefit.continuing.heading.obligated')}
+          labelSize="h3"
+          value={this.props.Obligated}
+          onValidate={this.handleValidation}
```
Comment (bmallred) [inline src/components/Section/Foreign/Activities/BenefitActivity/ContinuingBenefit.jsx]: The other fields are using `this.props.onValidate`

File: src/components/Section/Foreign/Activities/BenefitActivity/FutureBenefit.jsx [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+            {...this.props.Value}
+            onUpdate={this.updateValue}
+            onValidate={this.props.onValidate}
+          />
+          <div className="flags">
+            <Checkbox name="ValueEstimated"
+              label={i18n.t('foreign.activities.benefit.future.label.valueEstimated')}
+              toggle="false"
+              checked={this.props.ValueEstimated}
+              onUpdate={this.updateValueEstimated}
```
Comment (bmallred) [inline src/components/Section/Foreign/Activities/BenefitActivity/FutureBenefit.jsx]: Needs `onValidate` assignment

File: src/components/Section/Foreign/Activities/BenefitActivity/FutureBenefit.jsx [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+            onValidate={this.props.onValidate}
+          />
+        </Field>
+
+        <Branch name="Obligated"
+          className="obligated"
+          label={i18n.t('foreign.activities.benefit.future.heading.obligated')}
+          labelSize="h3"
+          value={this.props.Obligated}
+          onValidate={this.handleValidation}
```
Comment (bmallred) [inline src/components/Section/Foreign/Activities/BenefitActivity/FutureBenefit.jsx]: Should it be `this.props.onValidate`?

File: src/components/Section/Foreign/Activities/BenefitActivity/OneTimeBenefit.jsx [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+            {...this.props.Value}
+            onUpdate={this.updateValue}
+            onValidate={this.props.onValidate}
+          />
+          <div className="flags">
+            <Checkbox name="ValueEstimated"
+              label={i18n.t('foreign.activities.benefit.oneTime.label.valueEstimated')}
+              toggle="false"
+              checked={this.props.ValueEstimated}
+              onUpdate={this.updateValueEstimated}
```
Comment (bmallred) [inline src/components/Section/Foreign/Activities/BenefitActivity/OneTimeBenefit.jsx]: Needs `onValidate`?

File: src/components/Section/Foreign/Activities/BenefitActivity/OneTimeBenefit.jsx [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+            onValidate={this.props.onValidate}
+          />
+        </Field>
+
+        <Branch name="Obligated"
+          className="obligated"
+          label={i18n.t('foreign.activities.benefit.oneTime.heading.obligated')}
+          labelSize="h3"
+          value={this.props.Obligated}
+          onValidate={this.handleValidation}
```
Comment (bmallred) [inline src/components/Section/Foreign/Activities/BenefitActivity/OneTimeBenefit.jsx]: `this.props.onValidate`

File: src/components/Section/Foreign/Activities/BenefitActivity/BenefitActivity.jsx (hunk 2 above)
Comment (floresj) [inline src/components/Section/Foreign/Activities/BenefitActivity/BenefitActivity.jsx]: We've done it with some of our other summaries.

File: src/components/Section/Foreign/Activities/BenefitActivity/ContinuingBenefit.jsx (hunk 3 above)
Comment (floresj) [inline src/components/Section/Foreign/Activities/BenefitActivity/ContinuingBenefit.jsx]: :+1:

File: src/components/Section/Foreign/Activities/BenefitActivity/BenefitActivity.jsx (hunk 2 above)
Comment (bmallred) [inline src/components/Section/Foreign/Activities/BenefitActivity/BenefitActivity.jsx]: I think the order came from @Jay-Neighbours-I72 to have strong emphasis on the summary and anything that gets floated right. If there are items missing this we need to fix it.

Comment (floresj) [inline src/components/Section/Foreign/Activities/BenefitActivity/BenefitActivity.jsx]: Ohhh I misread that. I thought you were asking why it was bolded but just realized it's looking at the non-bolded element. :+1:

**Pull request merged**